addons/account_payment/models/account_move.py

```diff
@@ -101,6 +102,8 @@ def action_view_payment_transactions(self):
         return action
 
     def _get_default_payment_link_values(self):
+        if not self.line_ids:
+            raise UserError(_("You can't register a payment because there is nothing left to pay on the selected journal items."))
```

Comment:
```suggestion
        if not self.line_ids:
            return {}
```
Not need to raise, just return as {}. It shows nothing to payment in the wizard.